cli/deployments/gitea.go

```diff
@@ -56,12 +56,26 @@ func (k Gitea) Describe() string {
 func (k Gitea) Delete(c *kubernetes.Cluster, ui *ui.UI) error {
 	ui.Note().Msg("Removing Gitea...")
 
+	message := "Deleting Gitea namespace " + GiteaDeploymentID
+	warning, err := helpers.WaitForCommandCompletion(ui, message,
+		func() (string, error) {
+			return c.DeleteNamespaceIfOwned(GiteaDeploymentID)
+		},
+	)
+	if err != nil {
+		return errors.Wrapf(err, "Failed deleting namespace %s", GiteaDeploymentID)
+	}
+	if warning != "" {
+		ui.Exclamation().Msg(warning)
+		return nil
```

Comment:
Same as above, why exit on warnings? A warning can be anything, this can result in unintended behavior.

Reply:
Added a comment and exiting only for a specific warning.